Kudu.Core/Functions/KedaFunctionTriggerProvider.cs

```diff
@@ -365,7 +365,7 @@ internal static bool TryGetWorkflowKedaTrigger(string hostJsonText, string appNa
                     break;
 
                 case TriggerTypes.Kafka:
-                    metadata["bootstrapServers"] = metadata["brokerList"];
+                    metadata["bootstrapServersFromEnv"] = metadata["brokerList"];
```

Comment:
Unit test?  also, add link to Keda scaler doc where `bootstrapServersFromEnv` is referenced

Reply:
FromEnv suffix is the general grammar.   I can't see the document, however, it has. https://github.com/kedacore/keda/blob/main/pkg/scalers/kafka_scaler.go#L96

Reply:
OK! Merging PR then!